src/SFA.DAS.AANHub.Application/Common/PutCommandResult.cs

```diff
@@ -0,0 +1,6 @@
+﻿using System.Diagnostics.CodeAnalysis;
+
+namespace SFA.DAS.AANHub.Application.Common;
+
+[ExcludeFromCodeCoverage]
+public record PutCommandResult(bool NewResourceCreated);
```

Comment:
I dont see a use of this at all. We should just return a `Unit` from the command instead

Reply:
File deleted and replaced with `SuccessCommandResult` as discussed